docs/product-highlights-and-benefits/live-context.mdx

```diff
@@ -1,65 +1,87 @@
 ---
-title: Workstream Context
-description: Powered by Workstream Pattern Engine, Workstream Context can now understand recent workstream contexts, eliminating the need for manual grounding.
+title: Live Context
+description: Powered by Workstream Pattern Engine, Live Context is the world’s first Temporally Grounded Copilot that understands your workflow.
 ---
 
+import {MiniSpacer} from "/src/components/Spacers";
 import Video from "/src/components/Video";
 
-# Applying Workstream Context to your Copilot
+# Applying Live Context to your Copilot
+
+:::info
+This feature is currently in Alpha and is now generally available to all users. It is only available to macOS & Windows users, with Linux support coming ASAP 🙏🏼.
+:::
+
 Pieces for Developers has thus far been a productivity tool that is integrated into all of your tools using plugins and extensions, and allows users to have deep contextual conversations when they manually add context from folders, files, snippets, etc. While this is useful and the users love it, we want to go one step further.
 
-We would like to introduce the Pieces for Developers **Workstream Pattern Engine**, which powers the world’s first **Temporally Grounded Copilot**. Our goal is to push the limits of intelligent Copilot interactions through truly horizontal context awareness across the operating system, enabling your copilot to understand what you've been working on and keep up with the productivity demands that developers deal with every day. Together with your help, the Pieces Copilot will become the first that can understand recent workstream contexts, eliminating the need for manual grounding.
+<Video type={'youtube'} src={'https://www.youtube.com/embed/aP8u95RTCGE?si=y4bhrmzG9KWsD1AS'}/>
+<MiniSpacer />
+
+We would like to introduce the Pieces for Developers **Workstream Pattern Engine (WPE)**, which powers the world’s first **Temporally Grounded Copilot**. Our goal is to push the limits of intelligent Copilot interactions through truly horizontal context awareness across the operating system, enabling your copilot to understand what you've been working on and keep up with the productivity demands that developers deal with every day. Together with your help, the Pieces Copilot will become the first that can understand recent context, eliminating the need for manual grounding.
```

Comment:
Maybe for this, we need to introduce Live Context as the feature so that when we refer to in in the subsequent paragraphs, ppl know what's up.

"We would like to introduce Live Context in Pieces for Developers, powered by our **Workstream Pattern Engine (WPE)**, which enables the world’s first **Temporally Grounded Copilot**"

Reply:
@mack-at-pieces Can you take another look at this, it has been updated from rosie

Reply:
Nice - I think we can still slide the "Live Context" part of the first sentence in here. Lmk thoughts.

Reply:
updated